controller/work_item_link.go

```diff
@@ -513,3 +518,14 @@ func ConvertLinkToModel(appLink app.WorkItemLinkSingle) (*link.WorkItemLink, err
 
 	return &modelLink, nil
 }
+
+func ValidateSingleParent(ctx *workItemLinkContext, targetID uint64, linkTypeID uuid.UUID) error {
+	result := link.WorkItemLink{}
+	var db *gorm.DB
+	db = db.Where("link_type_id=? AND target_id=?", linkTypeID, targetID).Find(&result)
```

Comment:
@DhritiShikhar To be clear, you are not adding any DB constraint, right? Rather you are going to add the restriction in the code, right?